### DISCUSSION TITLE
2008/05/07/labview-%E9%9D%A2%E5%90%91%E5%AF%B9%E8%B1%A1%E7%A8%8B%E5%BA%8F%E8%AE%BE%E8%AE%A1%E7%9A%84%E7%AE%80%E4%BB%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#218] [Announcements]
# 2008/05/07/labview-%E9%9D%A2%E5%90%91%E5%AF%B9%E8%B1%A1%E7%A8%8B%E5%BA%8F%E8%AE%BE%E8%AE%A1%E7%9A%84%E7%AE%80%E4%BB%8B

LabVIEW 的数据流驱动模式，与面向过程的编程思想有些类似。它们都是把程序看成是一组过程或功能的集合，LabVIEW 利用数据流控制这些功能执行的顺序。由于开发者可以随意的修改、调用这些功能模块，在程序开发的后段，模块之间的划分会变得模糊，依赖关系也变得无序。这种方式就不再适合大型程序的开发。

https://blog.qizhen.xyz/2008/05/07/labview-%E9%9D%A2%E5%90%91%E5%AF%B9%E8%B1%A1%E7%A8%8B%E5%BA%8F%E8%AE%BE%E8%AE%A1%E7%9A%84%E7%AE%80%E4%BB%8B

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Boone
2011年05月16日 14:53 编辑
我也在拜读中，Ruan有空欢迎来QQ（LabVIEW_Home 13267790）群指点一下.

回复
Boone
2011年05月16日 19:52 编辑
Sorry to bother you!

回复
ruanqizhen
2011年05月17日 13:42 编辑
我没有qq，因为公司不能用。如果有啥问题的话可以写在这里，我看看能否帮上忙。

回复
竹贤
2008年05月10日 06:05 编辑
请大侠再稍讲得详细一点。

回复
arrowpig
2008年05月9日 19:14 编辑
你名气可真大，这次回苏州，碰到我一个老乡，问我，认识不认识一个叫阮奇祯的人，说他的一个同学被你加为好友了

回复
Zhen
2008年05月7日 21:37 编辑
领会中…
公司还在用7.1，8.5正在购买中…
